### PR TITLE
Add set timer and check time feature for focus timer

### PR DESCRIPTION
**Pull request by JingHaoooo**
satisfies #63  and #68

### CODE REVIEW

File: src/main/java/parser/FocusCommandParser.java [hunk 1]
```diff
@@ -1,11 +1,7 @@
 package parser;
 
 import commands.Command;
-import commands.focuscommands.SetResumeCommand;
-import commands.focuscommands.StartTimerCommand;
-import commands.focuscommands.StopTimerCommand;
-import commands.focuscommands.SwitchTimerCommand;
-import commands.focuscommands.SetPauseCommand;
+import commands.focuscommands.*;
```
Comment (genexus85) [inline src/main/java/parser/FocusCommandParser.java:4]: Imported classes should always be listed explicitly, do ensure to list out the individual imports.

File: src/main/java/commands/focuscommands/SetTimingCommand.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import focus.FocusTimer;
+
+public class SetTimingCommand implements Command {
+    private FocusTimer focusTimer;
+    private int desiredDuration;
+
+    public SetTimingCommand(FocusTimer timer, String userInput) throws FocusException {
+        this.focusTimer = timer;
+        String[] parts = userInput.trim().split("\\s", 2);
+        try {
```
Comment (genexus85) [inline src/main/java/commands/focuscommands/SetTimingCommand.java:14]: Perhaps you could add a space before the try-catch block to improve readability

**Review by genexus85 — COMMENTED** [review]
Overall, looks good to me, just to nitpick on certain areas as highlighted in the review

File: src/main/java/parser/FocusCommandParser.java (hunk 1 above)
Comment (JingHaoooo) [reply src/main/java/parser/FocusCommandParser.java:4]: Thanks for the suggestion! Will make the required changes.

File: src/main/java/commands/focuscommands/SetTimingCommand.java (hunk 2 above)
Comment (JingHaoooo) [reply src/main/java/commands/focuscommands/SetTimingCommand.java:14]: Got it!

**Pull request merged**